### ISSUE TITLE
Text failed. "fillNumberInput" failed.

### ISSUE THREAD

**Issue by FiftyTwoSouth** [Brismuth-Apps-LLC/bulk-texter-lite#211]
Using v.2.0.10 

Tried multiple times in both Chrome and Firefox without success. I see from previous posts it could be the result of a UI tweak on Google's side? Please let me know if I can provide more info.

**Comment by brismuth** [comment]
Is this happening for every phone number you add? For example, if you simply put your own phone number in the "numbers, names" input, does it still happen or does it send the message?

**Comment by FiftyTwoSouth** [comment]
Yes. Tried with names, without names. With varying number formats. Every time, once I click Send Messages, nothing happens until it errors out.

**Comment by brismuth** [comment]
It could be that Google is making changes to the interface that are keeping it from working for you.

Alternatively (this is a shot in the dark, based on another user running into it the other day), can you confirm that the Google Voice UI is set to English for you? Bulk Texter Lite does not currently support other languages for the Google Voice interface.

**Comment by FiftyTwoSouth** [comment]
Yep, definitely set to English. Good thought though. Anything else I can try?

**Comment by FiftyTwoSouth** [comment]
Fixed itself! Thanks for your help!

**Comment by brismuth** [comment]
Weird! But good for you, I guess, haha. 

Are you sure you can’t think of anything you changed? A few people have run into this issue, I’d love to know what else to suggest or fix. :)

**Comment by FiftyTwoSouth** [comment]
I tried it on two accounts. The first one I'd just migrated to Workspace from a legacy G-Suite account, and the new subscription doesn't include Voice, which to begin with is weird to not include in the paid subscription a service that's included in a free @gmail account, but also, instead of clearly stating that the service was no longer functional, it loaded the page like normal, but just made the + button unresponsive. Took me 45 min with support to figure out what was going on. So I'll say that's mostly on Google and a little on me. Problem was def not with your software.

The second account just worked when I tried it today, though it was on a different computer, but I swear I got the same error multiple times on that account previously. So either I was moving too fast and thought I'd changed accounts when I hadn't, or it just started working. 

Regardless, thanks so much for your quick responses. It's really appreciated.
